submodules/eks/main.tf

```diff
@@ -3,6 +3,7 @@ data "aws_caller_identity" "aws_account" {}
 
 locals {
   eks_cluster_name  = var.deploy_id
+  vpc_id            = var.vpc_id
```

Comment:
if these are the same, why not just use `var.vpc_id`

Reply:
done